tests/data/semparse/type_declarations/type_declaration_test.py

```diff
@@ -8,3 +11,57 @@ def test_basic_types_conflict_on_names(self):
         type_a = types.NamedBasicType("A")
         type_b = types.NamedBasicType("B")
         assert type_a.resolve(type_b) is None
+
+    def test_valid_actions(self):
```

Comment:
Can you change all of these from `test_valid_actions` to `test_get_valid_actions`?

Reply:
Done.